server.js

```diff
@@ -16,11 +16,90 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+const { Schema } = mongoose;
+const PostSchema = new Schema({
+  text: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+    trim: true
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now
+  }
+})
+
+const Post = mongoose.model("Post", PostSchema)
+
 // Start defining your routes here
 app.get("/", (req, res) => {
   res.send("Hello Technigo!");
```

Comment:
I would possibly have changed this message to something more related to the project, in case a user goes to this endpoint